tests/end-to-end/api/01-users.js

```diff
@@ -16,6 +16,47 @@ import { customFieldText, clearCustomFields, setCustomFields } from '../../data/
 import { updatePermission, updateSetting } from '../../data/permissions.helper';
 import { createUser, login } from '../../data/users.helper.js';
 
+function createTestUser() {
+	return new Promise((resolve) => {
+		const username = `user.test.${ Date.now() }`;
+		const email = `${ username }@rocket.chat`;
+		request.post(api('users.create'))
+			.set(credentials)
+			.send({ email, name: username, username, password })
+			.end((err, res) => resolve(res.body.user));
+	});
+}
+
+function loginTestUser(user) {
```

Comment:
Same as above here. We have a login helper [here](https://github.com/RocketChat/Rocket.Chat/blob/develop/tests/data/users.helper.js#L13).

Reply:
Using helper code now here as well